commcare/src/main/java/org/motechproject/commcare/service/CommcareApplicationService.java

```diff
@@ -0,0 +1,18 @@
+package org.motechproject.commcare.service;
+
+import org.motechproject.commcare.domain.CommcareApplicationJson;
+
+import java.util.List;
+
+/**
+ * A service to CommcareApplicationService is responsible for retriving applications by configuration name
+ */
+public interface CommcareApplicationService {
```

Comment:
Merge this new method into CommcareSchemaService - we do not need a new service for this feature